### DISCUSSION TITLE
Cannot reply the message to Facebook

### DISCUSSION THREAD

**Discussion by waranon** [chatwoot/chatwoot#6650] [Self hosted]
Able to receive the message from facebook but cannot reply. 

`srv-captain--chatx-worker.1.sqxibpj0doiw@caprover1101onubuntu2004-s-1vcpu-2gb-intel-sgp1-01    | E, [2023-03-10T17:27:45.961263 #1] ERROR -- : [ActiveJob] [SendReplyJob] [dd40f059-e62d-412a-9d8c-e5f96e277cfc] Invalid appsecret_proof provided in the API argument (subcode: )
srv-captain--chatx-worker.1.sqxibpj0doiw@caprover1101onubuntu2004-s-1vcpu-2gb-intel-sgp1-01    | I, [2023-03-10T17:27:45.962180 #1]  INFO -- : [ActiveJob] [SendReplyJob] [dd40f059-e62d-412a-9d8c-e5f96e277cfc] Performed SendReplyJob (Job ID: dd40f059-e62d-412a-9d8c-e5f96e277cfc) from Sidekiq(high) in 356.85ms
`

Above is an error from worker when send reply message to facebook and also no discussion about this here.

**Comment by waranon** [comment]
<img width="969" alt="image" src="https://user-images.githubusercontent.com/18227309/224386667-c9180314-b831-454a-a116-7124a1bad596.png">

all option unchecked under App settings > Advance

**Reply by waranon** [reply]
login dialog use fb api version v14 while Meta app dashboard has only v16.0

**Reply by waranon** [reply]
@pranavrajs Can you help find out?

**Reply by pranavrajs** [reply]
https://stackoverflow.com/questions/18683421/why-do-i-get-invalid-appsecret-proof-provided-in-the-api-argument
Can you please check if the app credentials are correct?

**Reply by pranavrajs** [reply]
@tejaswinichile 👆
